internal/templating/main.go

```diff
@@ -7,9 +7,43 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/natesales/bcg/internal/config"
 	"github.com/joomcode/errorx"
 )
 
+type Neighbor struct {
+	Address string
+	Type string
+	Protocols []string
+}
+
+func neighborsToStruct(Neighbors []string, mp bool) []Neighbor {
+	neighbors := []Neighbor{}
+	for _, neighbor := range Neighbors {
+		// neighbor_type of neighbor, used when constructing session string PEER1vNEIGHBOR_TYPE
+		neighbor_type := "MP"
```

Comment:
I'd prefer this to be "46" since MP could mean there are other BGP AFIs like flowspec that may be supported later on.

Reply:
ah will do